### PR TITLE
Added highlight for unavailable games

### PR DESCRIPTION
**Pull request by poixen**
unavailable rooms

### CODE REVIEW

**Comment by Daenyth** [comment]
I'm not sure I like this.

Can we try adding an alternating color mask with a similar color difference from the white and grey background? This makes them a big block

File: cockatrice/src/gamesmodel.cpp [hunk 1]
```diff
@@ -79,12 +79,14 @@ QVariant GamesModel::data(const QModelIndex &index, int role) const
...
     if (role == Qt::UserRole)
         return index.row();
-    if (role != Qt::DisplayRole && role != SORT_ROLE && role != Qt::DecorationRole && role != Qt::TextAlignmentRole)
+    if (role != Qt::DisplayRole && role != SORT_ROLE && role != Qt::DecorationRole && role != Qt::TextAlignmentRole && role != Qt::BackgroundRole)
         return QVariant();
     if ((index.row() >= gameList.size()) || (index.column() >= columnCount()))
         return QVariant();
 
     const ServerInfo_Game &g = gameList[index.row()];
+    if (role == Qt::BackgroundRole && (g.player_count() == g.max_players()))
```
Comment (Daenyth) [inline cockatrice/src/gamesmodel.cpp]: The game can be unavailable when it's not just full players. Have a look at what the unavailable filter does.

**Comment by poixen** [comment]
I had already tried to to that, it was my initial idea, This was the outcome:
untitled

The code was: 
`return index.row() % 2 ? darkcolor : lightcolor;`

Not sure why it came out like that, it bugs out alot too.

**Comment by woogerboy21** [comment]
i like the idea.  not so sure i like the color choice but can't say there would be an alternate color to suggest.

**Comment by Nightfirecat** [comment]
I agree with @woogerboy21 here - the idea is nice, but I'm not so fond of the color theme...

Perhaps just using a more grey-heavy (*maybe* red-tinged) color to indicate full rooms would look better? Not sure if that'd also alleviate the need for zebra-striping.

**Comment by tooomm** [comment]
Have you tried just changing the text color for the unavailable games lines to some grey instead of black? That they look more faded out?
The red color background might be highlighting it over the actual available games. ;)

Maybe that helps with the background issue, too.

**Comment by poixen** [comment]
This might work. I like the text fading idea. Will try it out.

**Pull request closed without merging**